### ISSUE TITLE
`ex-serialization1-processor` segfaults sometimes

### ISSUE THREAD

**Issue by dennisklein** [FairRootGroup/FairRoot#1130]
During preparation of FairSoft `apr21p2` the test `ex_MQ_serialization1` failed in about ~10% of cases (randomly on different linux flavors and gcc versions, see [list of tested system](https://github.com/FairRootGroup/FairSoft/blob/apr21_patches/legacy/README.md#tested-systems)):

```
[ex-serialization1-processor][09:09:19.868628][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
Error in <TExMap::Remove>: key 1949127801 not found at 286
Warning in <TMessage::CheckObject>: reference to object of unavailable class TObject, offset=1949127801 pointer will be 0

 *** Break *** segmentation violation
 Generating stack trace...
 0x00007fc214ae8da2 in fair::mq::Device::RunWrapper() at /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/Source/fairmq/fairmq/Device.cxx:472 (discriminator 6) from /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/lib/libFairMQ.so.1.4
 0x00007fc214aeae89 in <unknown> from /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/lib/libFairMQ.so.1.4
 0x00007fc21158c36d in boost::detail::function::void_function_obj_invoker1<std::function<void (fair::mq::State)>, void, fair::mq::State>::invoke(boost::detail::function::function_buffer&, fair::mq::State) at /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/include/boost/function/function_template.hpp:159 from /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/lib/libFairMQStateMachine.so.1.4
 0x00007fc211595489 in boost::signals2::detail::signal_impl<void (fair::mq::State), boost::signals2::optional_last_value<void>, int, std::less<int>, boost::function<void (fair::mq::State)>, boost::function<void (boost::signals2::connection const&, fair::mq::State)>, boost::signals2::mutex>::operator()(fair::mq::State) at /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/include/boost/optional/optional.hpp:340 from /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/lib/libFairMQStateMachine.so.1.4
 0x00007fc21159aa7f in fair::mq::fsm::Machine_::ProcessWork() at /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/include/boost/signals2/detail/signal_template.hpp:726 from /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/lib/libFairMQStateMachine.so.1.4
 0x00007fc21158ab8e in fair::mq::StateMachine::ProcessWork() at /usr/include/c++/8/bits/std_function.h:256 from /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/lib/libFairMQStateMachine.so.1.4
 0x00007fc214af75ad in fair::mq::DeviceRunner::Run() at /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/Source/fairmq/fairmq/PluginManager.h:86 from /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/lib/libFairMQ.so.1.4
 0x000000000041911b in <unknown> from /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/FairRoot_v18.6_patches/build/examples/MQ/serialization/ex-serialization1-processor
 0x00007fc20a712493 in __libc_start_main + 0xf3 from /lib64/libc.so.6
 0x000000000041918e in <unknown> from /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/FairRoot_v18.6_patches/build/examples/MQ/serialization/ex-serialization1-processor
```

<details>
  <summary>full test log</summary>
 
```
System during compilation: Rocky Linux release 8.5 (Green Obsidian)
                           x86_64
System now               : Rocky Linux release 8.5 (Green Obsidian)
                           x86_64

RooFit v3.60 -- Developed by Wouter Verkerke and David Kirkby 
                Copyright (C) 2000-2013 NIKHEF, University of California & Stanford University
                All rights reserved, please read http://roofit.sourceforge.net/license.txt

[INFO] Bunch number 1/100 (1 %). Number of generated digis: 1652, payload = 46256 bytes
[INFO] Bunch number 2/100 (2 %). Number of generated digis: 1581, payload = 44268 bytes
[INFO] Bunch number 3/100 (3 %). Number of generated digis: 1554, payload = 43512 bytes
[INFO] Bunch number 4/100 (4 %). Number of generated digis: 1593, payload = 44604 bytes
[INFO] Bunch number 5/100 (5 %). Number of generated digis: 1627, payload = 45556 bytes
[INFO] Bunch number 6/100 (6 %). Number of generated digis: 1592, payload = 44576 bytes
[INFO] Bunch number 7/100 (7 %). Number of generated digis: 1589, payload = 44492 bytes
[INFO] Bunch number 8/100 (8 %). Number of generated digis: 1596, payload = 44688 bytes
[INFO] Bunch number 9/100 (9 %). Number of generated digis: 1572, payload = 44016 bytes
[INFO] Bunch number 10/100 (10 %). Number of generated digis: 1600, payload = 44800 bytes
[INFO] Bunch number 11/100 (11 %). Number of generated digis: 1588, payload = 44464 bytes
[INFO] Bunch number 12/100 (12 %). Number of generated digis: 1576, payload = 44128 bytes
[INFO] Bunch number 13/100 (13 %). Number of generated digis: 1607, payload = 44996 bytes
[INFO] Bunch number 14/100 (14 %). Number of generated digis: 1554, payload = 43512 bytes
[INFO] Bunch number 15/100 (15 %). Number of generated digis: 1612, payload = 45136 bytes
[INFO] Bunch number 16/100 (16 %). Number of generated digis: 1600, payload = 44800 bytes
[INFO] Bunch number 17/100 (17 %). Number of generated digis: 1548, payload = 43344 bytes
[INFO] Bunch number 18/100 (18 %). Number of generated digis: 1598, payload = 44744 bytes
[INFO] Bunch number 19/100 (19 %). Number of generated digis: 1589, payload = 44492 bytes
[INFO] Bunch number 20/100 (20 %). Number of generated digis: 1541, payload = 43148 bytes
[INFO] Bunch number 21/100 (21 %). Number of generated digis: 1627, payload = 45556 bytes
[INFO] Bunch number 22/100 (22 %). Number of generated digis: 1584, payload = 44352 bytes
[INFO] Bunch number 23/100 (23 %). Number of generated digis: 1568, payload = 43904 bytes
[INFO] Bunch number 24/100 (24 %). Number of generated digis: 1604, payload = 44912 bytes
[INFO] Bunch number 25/100 (25 %). Number of generated digis: 1622, payload = 45416 bytes
[INFO] Bunch number 26/100 (26 %). Number of generated digis: 1571, payload = 43988 bytes
[INFO] Bunch number 27/100 (27 %). Number of generated digis: 1589, payload = 44492 bytes
[INFO] Bunch number 28/100 (28 %). Number of generated digis: 1552, payload = 43456 bytes
[INFO] Bunch number 29/100 (29 %). Number of generated digis: 1575, payload = 44100 bytes
[INFO] Bunch number 30/100 (30 %). Number of generated digis: 1587, payload = 44436 bytes
[INFO] Bunch number 31/100 (31 %). Number of generated digis: 1614, payload = 45192 bytes
[INFO] Bunch number 32/100 (32 %). Number of generated digis: 1609, payload = 45052 bytes
[INFO] Bunch number 33/100 (33 %). Number of generated digis: 1620, payload = 45360 bytes
[INFO] Bunch number 34/100 (34 %). Number of generated digis: 1579, payload = 44212 bytes
[INFO] Bunch number 35/100 (35 %). Number of generated digis: 1602, payload = 44856 bytes
[INFO] Bunch number 36/100 (36 %). Number of generated digis: 1587, payload = 44436 bytes
[INFO] Bunch number 37/100 (37 %). Number of generated digis: 1557, payload = 43596 bytes
[INFO] Bunch number 38/100 (38 %). Number of generated digis: 1523, payload = 42644 bytes
[INFO] Bunch number 39/100 (39 %). Number of generated digis: 1645, payload = 46060 bytes
[INFO] Bunch number 40/100 (40 %). Number of generated digis: 1622, payload = 45416 bytes
[INFO] Bunch number 41/100 (41 %). Number of generated digis: 1614, payload = 45192 bytes
[INFO] Bunch number 42/100 (42 %). Number of generated digis: 1599, payload = 44772 bytes
[INFO] Bunch number 43/100 (43 %). Number of generated digis: 1615, payload = 45220 bytes
[INFO] Bunch number 44/100 (44 %). Number of generated digis: 1578, payload = 44184 bytes
[INFO] Bunch number 45/100 (45 %). Number of generated digis: 1585, payload = 44380 bytes
[INFO] Bunch number 46/100 (46 %). Number of generated digis: 1546, payload = 43288 bytes
[INFO] Bunch number 47/100 (47 %). Number of generated digis: 1650, payload = 46200 bytes
[INFO] Bunch number 48/100 (48 %). Number of generated digis: 1591, payload = 44548 bytes
[INFO] Bunch number 49/100 (49 %). Number of generated digis: 1573, payload = 44044 bytes
[INFO] Bunch number 50/100 (50 %). Number of generated digis: 1627, payload = 45556 bytes
[INFO] Bunch number 51/100 (51 %). Number of generated digis: 1573, payload = 44044 bytes
[INFO] Bunch number 52/100 (52 %). Number of generated digis: 1597, payload = 44716 bytes
[INFO] Bunch number 53/100 (53 %). Number of generated digis: 1614, payload = 45192 bytes
[INFO] Bunch number 54/100 (54 %). Number of generated digis: 1560, payload = 43680 bytes
[INFO] Bunch number 55/100 (55 %). Number of generated digis: 1576, payload = 44128 bytes
[INFO] Bunch number 56/100 (56 %). Number of generated digis: 1577, payload = 44156 bytes
[INFO] Bunch number 57/100 (57 %). Number of generated digis: 1597, payload = 44716 bytes
[INFO] Bunch number 58/100 (58 %). Number of generated digis: 1586, payload = 44408 bytes
[INFO] Bunch number 59/100 (59 %). Number of generated digis: 1617, payload = 45276 bytes
[INFO] Bunch number 60/100 (60 %). Number of generated digis: 1559, payload = 43652 bytes
[INFO] Bunch number 61/100 (61 %). Number of generated digis: 1648, payload = 46144 bytes
[INFO] Bunch number 62/100 (62 %). Number of generated digis: 1595, payload = 44660 bytes
[INFO] Bunch number 63/100 (63 %). Number of generated digis: 1570, payload = 43960 bytes
[INFO] Bunch number 64/100 (64 %). Number of generated digis: 1603, payload = 44884 bytes
[INFO] Bunch number 65/100 (65 %). Number of generated digis: 1615, payload = 45220 bytes
[INFO] Bunch number 66/100 (66 %). Number of generated digis: 1578, payload = 44184 bytes
[INFO] Bunch number 67/100 (67 %). Number of generated digis: 1599, payload = 44772 bytes
[INFO] Bunch number 68/100 (68 %). Number of generated digis: 1673, payload = 46844 bytes
[INFO] Bunch number 69/100 (69 %). Number of generated digis: 1600, payload = 44800 bytes
[INFO] Bunch number 70/100 (70 %). Number of generated digis: 1612, payload = 45136 bytes
[INFO] Bunch number 71/100 (71 %). Number of generated digis: 1620, payload = 45360 bytes
[INFO] Bunch number 72/100 (72 %). Number of generated digis: 1587, payload = 44436 bytes
[INFO] Bunch number 73/100 (73 %). Number of generated digis: 1647, payload = 46116 bytes
[INFO] Bunch number 74/100 (74 %). Number of generated digis: 1589, payload = 44492 bytes
[INFO] Bunch number 75/100 (75 %). Number of generated digis: 1609, payload = 45052 bytes
[INFO] Bunch number 76/100 (76 %). Number of generated digis: 1599, payload = 44772 bytes
[INFO] Bunch number 77/100 (77 %). Number of generated digis: 1609, payload = 45052 bytes
[INFO] Bunch number 78/100 (78 %). Number of generated digis: 1606, payload = 44968 bytes
[INFO] Bunch number 79/100 (79 %). Number of generated digis: 1632, payload = 45696 bytes
[INFO] Bunch number 80/100 (80 %). Number of generated digis: 1634, payload = 45752 bytes
[INFO] Bunch number 81/100 (81 %). Number of generated digis: 1608, payload = 45024 bytes
[INFO] Bunch number 82/100 (82 %). Number of generated digis: 1608, payload = 45024 bytes
[INFO] Bunch number 83/100 (83 %). Number of generated digis: 1603, payload = 44884 bytes
[INFO] Bunch number 84/100 (84 %). Number of generated digis: 1605, payload = 44940 bytes
[INFO] Bunch number 85/100 (85 %). Number of generated digis: 1598, payload = 44744 bytes
[INFO] Bunch number 86/100 (86 %). Number of generated digis: 1611, payload = 45108 bytes
[INFO] Bunch number 87/100 (87 %). Number of generated digis: 1638, payload = 45864 bytes
[INFO] Bunch number 88/100 (88 %). Number of generated digis: 1612, payload = 45136 bytes
[INFO] Bunch number 89/100 (89 %). Number of generated digis: 1586, payload = 44408 bytes
[INFO] Bunch number 90/100 (90 %). Number of generated digis: 1606, payload = 44968 bytes
[INFO] Bunch number 91/100 (91 %). Number of generated digis: 1557, payload = 43596 bytes
[INFO] Bunch number 92/100 (92 %). Number of generated digis: 1624, payload = 45472 bytes
[INFO] Bunch number 93/100 (93 %). Number of generated digis: 1624, payload = 45472 bytes
[INFO] Bunch number 94/100 (94 %). Number of generated digis: 1601, payload = 44828 bytes
[INFO] Bunch number 95/100 (95 %). Number of generated digis: 1586, payload = 44408 bytes
[INFO] Bunch number 96/100 (96 %). Number of generated digis: 1632, payload = 45696 bytes
[INFO] Bunch number 97/100 (97 %). Number of generated digis: 1571, payload = 43988 bytes
[INFO] Bunch number 98/100 (98 %). Number of generated digis: 1590, payload = 44520 bytes
[INFO] Bunch number 99/100 (99 %). Number of generated digis: 1609, payload = 45052 bytes
[INFO] Bunch number 100/100 (100 %). Number of generated digis: 1594, payload = 44632 bytes
[INFO] Data generation successful


RooFit v3.60 -- Developed by Wouter Verkerke and David Kirkby 
RooFit v3.60 -- Developed by Wouter Verkerke and David Kirkby 
                Copyright (C) 2000-2013 NIKHEF, University of California & Stanford University
                Copyright (C) 2000-2013 NIKHEF, University of California & Stanford University
                All rights reserved, please read http://roofit.sourceforge.net/license.txt
                All rights reserved, please read http://roofit.sourceforge.net/license.txt


[ex-serialization1-sampler][09:09:13.851885][INFO][DeviceRunner.cxx:67:HandleGeneralOptions] 
      ______      _    _______  _________ 
     / ____/___ _(_)_______   |/  /_  __ \    version 1.4.44
    / /_  / __ `/ / ___/__  /|_/ /_  / / /    build RelWithDebInfo
   / __/ / /_/ / / /    _  /  / / / /_/ /     https://github.com/FairRootGroup/FairMQ
  /_/    \__,_/_/_/     /_/  /_/  \___\_\     LGPL-3.0  © 2012-2021 GSI

[ex-serialization1-sink][09:09:13.851885][INFO][DeviceRunner.cxx:67:HandleGeneralOptions] 
      ______      _    _______  _________ 
     / ____/___ _(_)_______   |/  /_  __ \    version 1.4.44
    / /_  / __ `/ / ___/__  /|_/ /_  / / /    build RelWithDebInfo
   / __/ / /_/ / / /    _  /  / / / /_/ /     https://github.com/FairRootGroup/FairMQ
  /_/    \__,_/_/_/     /_/  /_/  \___\_\     LGPL-3.0  © 2012-2021 GSI

[ex-serialization1-sink][09:09:13.852094][STATE][StateMachine.cxx:94:on_entry] Starting FairMQ state machine --> IDLE
[ex-serialization1-sampler][09:09:13.852094][STATE][StateMachine.cxx:94:on_entry] Starting FairMQ state machine --> IDLE
[ex-serialization1-sink][09:09:13.852386][STATE][StateMachine.cxx:194:ProcessWork] IDLE ---> INITIALIZING DEVICE
[ex-serialization1-sampler][09:09:13.852386][STATE][StateMachine.cxx:194:ProcessWork] IDLE ---> INITIALIZING DEVICE
[ex-serialization1-sink][09:09:14.000571][STATE][StateMachine.cxx:194:ProcessWork] INITIALIZING DEVICE ---> INITIALIZED
[ex-serialization1-sink][09:09:14.000667][STATE][StateMachine.cxx:194:ProcessWork] INITIALIZED ---> BINDING
[ex-serialization1-sink][09:09:14.002103][STATE][StateMachine.cxx:194:ProcessWork] BINDING ---> BOUND
[ex-serialization1-sink][09:09:14.002190][STATE][StateMachine.cxx:194:ProcessWork] BOUND ---> CONNECTING
[ex-serialization1-sink][09:09:14.002205][STATE][StateMachine.cxx:194:ProcessWork] CONNECTING ---> DEVICE READY
[ex-serialization1-sink][09:09:14.002231][STATE][StateMachine.cxx:194:ProcessWork] DEVICE READY ---> INITIALIZING TASK
[ex-serialization1-sink][09:09:14.002244][STATE][StateMachine.cxx:194:ProcessWork] INITIALIZING TASK ---> READY
[ex-serialization1-sink][09:09:14.002268][STATE][StateMachine.cxx:194:ProcessWork] READY ---> RUNNING
[ex-serialization1-sink][09:09:14.002279][INFO][Device.cxx:438:RunWrapper] fair::mq::Device running...
[ex-serialization1-sampler][09:09:14.083928][STATE][StateMachine.cxx:194:ProcessWork] INITIALIZING DEVICE ---> INITIALIZED
[ex-serialization1-sampler][09:09:14.084054][STATE][StateMachine.cxx:194:ProcessWork] INITIALIZED ---> BINDING
[ex-serialization1-sampler][09:09:14.085185][STATE][StateMachine.cxx:194:ProcessWork] BINDING ---> BOUND
[ex-serialization1-sampler][09:09:14.085277][STATE][StateMachine.cxx:194:ProcessWork] BOUND ---> CONNECTING
[ex-serialization1-sampler][09:09:14.085292][STATE][StateMachine.cxx:194:ProcessWork] CONNECTING ---> DEVICE READY
[ex-serialization1-sampler][09:09:14.085324][STATE][StateMachine.cxx:194:ProcessWork] DEVICE READY ---> INITIALIZING TASK
[ex-serialization1-sampler][09:09:14.085347][STATE][StateMachine.cxx:194:ProcessWork] INITIALIZING TASK ---> READY
[ex-serialization1-sampler][09:09:14.085363][STATE][StateMachine.cxx:194:ProcessWork] READY ---> RUNNING
[ex-serialization1-sampler][09:09:14.085373][INFO][Device.cxx:438:RunWrapper] fair::mq::Device running...
[ex-serialization1-sampler][09:09:14.085455][INFO][Ex1Sampler.h:55:Run] Number of events to process: 100
[ex-serialization1-sampler][09:09:14.187807][INFO][Ex1Sampler.h:68:Run] Sent 100 messages!
[ex-serialization1-sampler][09:09:14.187914][INFO][Device.cxx:746:LogSocketRates] data1[0]: in: 0 (0 MB) out: 980.392 (129.506 MB)
[ex-serialization1-sampler][09:09:14.188023][STATE][StateMachine.cxx:194:ProcessWork] RUNNING ---> READY
[ex-serialization1-sampler][09:09:14.188129][STATE][StateMachine.cxx:194:ProcessWork] READY ---> RESETTING TASK
[ex-serialization1-sampler][09:09:14.188143][STATE][StateMachine.cxx:194:ProcessWork] RESETTING TASK ---> DEVICE READY
[ex-serialization1-sampler][09:09:14.188161][STATE][StateMachine.cxx:194:ProcessWork] DEVICE READY ---> RESETTING DEVICE
[ex-serialization1-sampler][09:09:14.188727][STATE][StateMachine.cxx:194:ProcessWork] RESETTING DEVICE ---> IDLE
[ex-serialization1-sampler][09:09:14.188806][STATE][StateMachine.cxx:194:ProcessWork] IDLE ---> EXITING
[ex-serialization1-sampler][09:09:14.253273][STATE][StateMachine.cxx:101:on_exit] Exiting FairMQ state machine
[ex-serialization1-sink][09:09:15.003011][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)

RooFit v3.60 -- Developed by Wouter Verkerke and David Kirkby 
                Copyright (C) 2000-2013 NIKHEF, University of California & Stanford University
                All rights reserved, please read http://roofit.sourceforge.net/license.txt

[ex-serialization1-processor][09:09:15.865208][INFO][DeviceRunner.cxx:67:HandleGeneralOptions] 
      ______      _    _______  _________ 
     / ____/___ _(_)_______   |/  /_  __ \    version 1.4.44
    / /_  / __ `/ / ___/__  /|_/ /_  / / /    build RelWithDebInfo
   / __/ / /_/ / / /    _  /  / / / /_/ /     https://github.com/FairRootGroup/FairMQ
  /_/    \__,_/_/_/     /_/  /_/  \___\_\     LGPL-3.0  © 2012-2021 GSI

[ex-serialization1-processor][09:09:15.865416][STATE][StateMachine.cxx:94:on_entry] Starting FairMQ state machine --> IDLE
[ex-serialization1-processor][09:09:15.865692][STATE][StateMachine.cxx:194:ProcessWork] IDLE ---> INITIALIZING DEVICE
[ex-serialization1-processor][09:09:15.866353][STATE][StateMachine.cxx:194:ProcessWork] INITIALIZING DEVICE ---> INITIALIZED
[ex-serialization1-processor][09:09:15.866437][STATE][StateMachine.cxx:194:ProcessWork] INITIALIZED ---> BINDING
[ex-serialization1-processor][09:09:15.866456][STATE][StateMachine.cxx:194:ProcessWork] BINDING ---> BOUND
[ex-serialization1-processor][09:09:15.866482][STATE][StateMachine.cxx:194:ProcessWork] BOUND ---> CONNECTING
[ex-serialization1-processor][09:09:15.867543][STATE][StateMachine.cxx:194:ProcessWork] CONNECTING ---> DEVICE READY
[ex-serialization1-processor][09:09:15.867626][STATE][StateMachine.cxx:194:ProcessWork] DEVICE READY ---> INITIALIZING TASK
[ex-serialization1-processor][09:09:15.867643][STATE][StateMachine.cxx:194:ProcessWork] INITIALIZING TASK ---> READY
[ex-serialization1-processor][09:09:15.867668][STATE][StateMachine.cxx:194:ProcessWork] READY ---> RUNNING
[ex-serialization1-processor][09:09:15.867680][INFO][Device.cxx:438:RunWrapper] fair::mq::Device running...
[ex-serialization1-sink][09:09:16.003244][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-processor][09:09:16.867985][INFO][Device.cxx:746:LogSocketRates] data1[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-processor][09:09:16.868041][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:17.003440][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-processor][09:09:17.868187][INFO][Device.cxx:746:LogSocketRates] data1[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-processor][09:09:17.868232][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:18.003611][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-processor][09:09:18.868384][INFO][Device.cxx:746:LogSocketRates] data1[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-processor][09:09:18.868436][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:19.003781][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-processor][09:09:19.868578][INFO][Device.cxx:746:LogSocketRates] data1[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-processor][09:09:19.868628][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
Error in <TExMap::Remove>: key 1949127801 not found at 286
Warning in <TMessage::CheckObject>: reference to object of unavailable class TObject, offset=1949127801 pointer will be 0

 *** Break *** segmentation violation
 Generating stack trace...
[ex-serialization1-sink][09:09:20.003958][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
 0x00007fc214ae8da2 in fair::mq::Device::RunWrapper() at /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/Source/fairmq/fairmq/Device.cxx:472 (discriminator 6) from /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/lib/libFairMQ.so.1.4
 0x00007fc214aeae89 in <unknown> from /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/lib/libFairMQ.so.1.4
 0x00007fc21158c36d in boost::detail::function::void_function_obj_invoker1<std::function<void (fair::mq::State)>, void, fair::mq::State>::invoke(boost::detail::function::function_buffer&, fair::mq::State) at /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/include/boost/function/function_template.hpp:159 from /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/lib/libFairMQStateMachine.so.1.4
 0x00007fc211595489 in boost::signals2::detail::signal_impl<void (fair::mq::State), boost::signals2::optional_last_value<void>, int, std::less<int>, boost::function<void (fair::mq::State)>, boost::function<void (boost::signals2::connection const&, fair::mq::State)>, boost::signals2::mutex>::operator()(fair::mq::State) at /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/include/boost/optional/optional.hpp:340 from /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/lib/libFairMQStateMachine.so.1.4
 0x00007fc21159aa7f in fair::mq::fsm::Machine_::ProcessWork() at /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/include/boost/signals2/detail/signal_template.hpp:726 from /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/lib/libFairMQStateMachine.so.1.4
 0x00007fc21158ab8e in fair::mq::StateMachine::ProcessWork() at /usr/include/c++/8/bits/std_function.h:256 from /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/lib/libFairMQStateMachine.so.1.4
 0x00007fc214af75ad in fair::mq::DeviceRunner::Run() at /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/Source/fairmq/fairmq/PluginManager.h:86 from /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/install/lib/libFairMQ.so.1.4
 0x000000000041911b in <unknown> from /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/FairRoot_v18.6_patches/build/examples/MQ/serialization/ex-serialization1-processor
 0x00007fc20a712493 in __libc_start_main + 0xf3 from /lib64/libc.so.6
 0x000000000041918e in <unknown> from /tmp/jenkins-FairRootGroup-FairSoft-apr21_patches-17.15s/build/FairRoot_v18.6_patches/build/examples/MQ/serialization/ex-serialization1-processor
[ex-serialization1-sink][09:09:21.004161][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:22.004343][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:23.004511][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:24.004675][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:25.004836][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:26.005023][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:27.005184][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:28.005350][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:29.005510][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:30.005671][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:31.005828][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:32.005987][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:33.006147][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:34.006306][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:35.006468][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:36.006629][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:37.006793][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:38.006954][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:39.007113][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:40.007273][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:41.007434][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)
[ex-serialization1-sink][09:09:42.007600][INFO][Device.cxx:746:LogSocketRates] data2[0]: in: 0 (0 MB) out: 0 (0 MB)

    Test #30: ex_MQ_serialization1 ...................***Timeout  30.02 sec
```
</details>

[full build log](https://cdash.gsi.de/testDetails.php?test=10383995&build=309281)

**Comment by dennisklein** [comment]
possibly duplicate or related to #1048, going to cherry-pick the fix in #1068 to `v18.4_patches` and `v18.6_patches` to see if it is related.

**Comment by dennisklein** [comment]
appears to be fixed by the backports
